docs/ocis/development/testing.md

```diff
@@ -90,6 +91,7 @@ If you want to work on a specific issue
     TEST_SERVER_URL=https://localhost:9200 \
     TEST_OCIS=true \
     OCIS_REVA_DATA_ROOT=/var/tmp/ocis/owncloud/ \
```

Comment:
Oh, this also needs `data/` in the `OCIS_REVA_DATA_ROOT`

Reply:
thank you for pointing that out, actually we should completely remove that line, because now we are running on ocis-storage and don't really use the `OCIS_REVA_DATA_ROOT`
the files are provisioned by webdav upload and deleted by the `DELETE_USER_DATA_CMD` command, so it should work without. I will try and adjust the PR

Reply:
I've deleted the variable in  844f03c799a3b506cb6daaa156bf0f7cc821b1de